### ISSUE TITLE
[Feature]: a station auto-play 

### ISSUE THREAD

**Issue by NOBELLIST** [fast4x/RiMusic#3959]
### Description

Some kind of station auto-play like the one in soundcloud when yourun out of music on a playlist i will give random songs instead of looping back to the first song in the playlist, effectively extending the playlist

### Suggested Solution

When the playlist ends, it will give other random songs tied with whatever genre or something to the last song. Kinda like soundclouds station thingy that you can toggle on & off. 

### Alternatives

The discover setting that is already in the app. 

### Additional Context

Basically what the station auto play is but for rimusic.
Also [station auto-play type count in this feature req : 3]
